x-pack/package.json

```diff
@@ -318,7 +318,6 @@
     "stats-lite": "^2.2.0",
     "style-it": "^2.1.3",
     "styled-components": "3.3.3",
-    "tar-fs": "1.13.0",
```

Comment:
This was used by the Phantom browser when Kibana supported that, but since it's no longer used, we can remove the dependency altogether.